src/gazebo_wind_plugin.cpp

```diff
@@ -134,13 +132,19 @@ void GazeboWindPlugin::OnUpdate(const common::UpdateInfo& _info) {
   }
 
   gazebo::msgs::Vector3d* force = new gazebo::msgs::Vector3d();
-  force->set_x(wind.X() + wind_gust.X());
-  force->set_y(wind.Y() + wind_gust.Y());
-  force->set_z(wind.Z() + wind_gust.Z());
+  force->set_x( wind_gust.X());
```

Comment:
Shouldn't this force be taking into account the drag model somehow? First time looking at this codebase...

Reply:
@jkflying Yeah I agree it would be nicer. This entire plugin needs an update since it does not include any derivatives, it just uses static lift and drag calculations.

Reply:
@jkflying Do you think we should also move gust to have wind velocity? I left the gust to act as force since we don't have the liftdrag plugin enabled for all multirotor models / models that don't have wings

Reply:
>  should also move gust to have wind velocity?

I think it would be better to keep them consistent, yes

Reply:
Would it be better if it is a separate PR or in this PR? basically this means we would need to add the lift drag plugin for all SITL models